### ISSUE TITLE
[Feature Request]audio_padding can set start time and end time independently

### ISSUE THREAD

**Issue by echoIIImk2** [Ajatt-Tools/mpvacious#98]
I hope audio_padding can set start time and end time independently, just like subs2srs does. Because I found that in most subtitles, start is accurate, adding a 50ms is enough, while end is often not accurate, need to add 300ms or 400ms.

**Comment by tatsumoto-ren** [comment]
Please read [issue.md](https://github.com/Ajatt-Tools/mpvacious/blob/master/.github/ISSUE_TEMPLATE/issue.md).

**Comment by tatsumoto-ren** [comment]
> in most subtitles, start is accurate, adding a 50ms is enough, while end is often not accurate, need to add 300ms or 400ms.

Can't you then set the timings manually? I imagine each subtitle is going to need different padding values.

**Comment by echoIIImk2** [comment]
What I mean is that start and end need to be set with different padding in most cases, so it is better to set them separately.

**Comment by tatsumoto-ren** [comment]
Yes, but how do you know ahead of time what padding you need to set? I think it's less error-prone to adjust the timings in mpv before making a card.
